doc/headers/modules.dot

```diff
@@ -22,12 +23,15 @@
   // log -> exec [color="black",fontsize=14,style="solid",fontname="Times-Roman"];
   exec -> ref [color="black",fontsize=14,style="solid",fontname="Times-Roman"];
   exec -> cuda[color="black",fontsize=14,style="solid",fontname="Times-Roman"];
+  exec -> hip[color="black",fontsize=14,style="solid",fontname="Times-Roman"];
   exec -> omp [color="black",fontsize=14,style="solid",fontname="Times-Roman"];
   omp -> linop [color="black",fontsize=14,style="solid",fontname="Times-Roman"];
   cuda -> linop [color="black",fontsize=14,style="solid",fontname="Times-Roman"];
+  hip -> linop [color="black",fontsize=14,style="solid",fontname="Times-Roman"];
   ref -> linop [color="black",fontsize=14,style="solid",fontname="Times-Roman"];
   omp -> stop [color="black",fontsize=14,style="solid",fontname="Times-Roman"];
   cuda -> stop [color="black",fontsize=14,style="solid",fontname="Times-Roman"];
+  hip -> linop [color="black",fontsize=14,style="solid",fontname="Times-Roman"];
```

Comment:
It looks to me like this edge is a duplicate. Did you maybe mean 
```suggestion
  hip -> stop [color="black",fontsize=14,style="solid",fontname="Times-Roman"];
```

Reply:
Actually, this is not used anymore, We could delete this file as well.